example/static_images.js

```diff
@@ -88,14 +88,23 @@ $(function() {
                 length: 10
             },
             decoder : {
-                readers : ["code_128_reader"]
+                readers : ["code_128_reader"],
+                drawScanline: true,
```

Comment:
This setting is only valid if ``numOfWorkers`` is set to ``0`` and does not do anything in this case. The drawing of the scanline is done in the ``Quagga.onProcessed`` below.